tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -31,4 +31,10 @@ Dependent Build should not Present
      Run Keyword IF     not ${isExist}        Log    Build attached to Build config has been deleted
      ...        ELSE    FAIL       Attached Build to Build config is not deleted
 
-
+Check Image Build Status
+  [Arguments]  ${target_status}     ${notebook_name}
+  ${build_status}=  Get Build Status    namespace=redhat-ods-applications  build_search_term=${notebook_name}
+  Run Keyword If    "${build_status}" == "Failed"
+  ...    Fail  The '${notebook_name}' image build has failed
+  ...  ELSE
+  ...    Should Be Equal    ${build_status}    ${target_status}
```

Comment:
I don't like this. If you have a `target_status`, the status could also be `Failed`, so why are you failing the TC directly?
I think you should check that `Should Be Equal    ${build_status}    ${target_status}` and let it fail if the two don't match.

Reply:
thanks..i only considered having notebook build creation , may be it will be useful for non functional testing in future. I have done modification